2023-10-14/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+)
+
+func concatenate(a string, b string) string {
+	return a + b
+}
+
+func main() {
+	fmt.Println(concatenate("Jo", "el"))
```

Comment:
When we call back this function we have to define the strings we want the function to output.